configuration-reference/functions/arrayconcatdouble.md

```diff
@@ -0,0 +1,25 @@
+---
+description: This section contains reference documentation for the arrayConcatDouble function.
+---
+
+# arrayConcatDouble
+
+Concatenates two arrays of doubles.
+
+## Signature
+
+> arrayConcatDouble('colName1', 'colName2')
+
+## Usage Examples
+
+This example assumes the multiValueTable columns mvCol1 and mvCol2 are both of type DOUBLE with
+singleValueField in the table schema set to false.
+
+
+```sql
+select mvCol1, 
+       arrayConcatInt(mvCol1, mvCol2) AS concatDoubles
```

Comment:
This should be `arrayConcatDouble`? Same for other pages

Reply:
Yes let me fix this, thanks!